src/tests/functional/navigation_tests.ts

```diff
@@ -144,6 +144,37 @@ export class NavigationTests extends TurboDriveTestCase {
 
     this.assert.equal(await this.pathname, "/src/tests/fixtures/frames/hello.html")
   }
+
+  async "test skip link with hash-only path scrolls to the anchor without a visit"() {
+    await this.clickSelector('a[href="#main"]')
+    await this.nextBody
```

Comment:
I think this asserts that the link click creates a visit (and therefore [changes the body](https://github.com/hotwired/turbo/blob/8386e599769d238d8b578453d344c94e3e28da3c/src/tests/helpers/turbo_drive_test_case.ts#L56)).

https://github.com/hotwired/turbo/pull/57/commits/d6eeead682bbf6700ec3b120ec284c97bf7d0df9 tests that clicking a same-page anchor does not reload the page.